libnavigation-ui/src/main/java/com/mapbox/navigation/ui/routealert/MapboxRouteAlert.kt

```diff
@@ -0,0 +1,108 @@
+package com.mapbox.navigation.ui.routealert
+
+import android.content.Context
+import androidx.annotation.ColorInt
+import com.mapbox.mapboxsdk.maps.Style
+import com.mapbox.mapboxsdk.style.layers.Property
+import com.mapbox.mapboxsdk.style.layers.PropertyFactory
+import com.mapbox.mapboxsdk.style.layers.PropertyValue
+import com.mapbox.navigation.base.trip.model.alert.RouteAlert
+import com.mapbox.navigation.base.trip.model.alert.RouteAlertType
+
+/**
+ * Default implementation to show different route alerts, refer to [RouteAlertType]
+ *
+ * @param context [Context] for retrieving route alert drawable
+ * @param style to add source/layer into
+ * @param options config which type of route alerts should be handled, refer to [RouteAlertOption]
+ */
+class MapboxRouteAlert(context: Context, style: Style, private val options: Int = Int.MAX_VALUE) {
```

Comment:
`options` must be one of `RouteAlertOption`, right? Should we avoid using default value not from the range of the `RouteAlertOption`?

```suggestion
class MapboxRouteAlert(context: Context, style: Style, private val options: Int) {
```

Reply:
this options can be combination of several `RouteAlertOption`, for example:
```kotlin
options = ROUTE_ALERT_COUNTRY_BORDER_CROSSING or ROUTE_ALERT_TOLL
```

Reply:
get it 👍 should add this as a sample to the kdoc?

Reply:
good idea 👍